server/config_test.go

```diff
@@ -15,6 +15,8 @@ func TestLoadUsers(t *testing.T) {
 		expUsers []user.UserWithRemoteIdentities
 		// userid -> plaintext password
 		expPasswds map[string]string
+
+		wantErr bool
```

Comment:
I think it's ok as is: if you *were* handling errors differently, than it would be important.